docs/getting-started/setup/using-bundlers.md

```diff
@@ -0,0 +1,116 @@
+---
+category: setup
+meta-title: Using bundlers | CKEditor 5 documentation
+order: 130
+modified_at: 2024-08-30
+---
+
+# Using bundlers to bundle CKEditor&nbsp;5
+
+This guide will show you how to configure popular bundlers like **Vite**, **esbuild**, **Rollup**, and **webpack** to build applications using CKEditor&nbsp;5. Showing how to set up or use these bundlers is not the goal of this guide.
+
+Note that CKEditor&nbsp;5 ships plain ES2022 JavaScript modules and CSS files. Therefore, with recent enough bundlers, you should be able to use CKEditor&nbsp;5 without any additional configuration. Or &ndash; in the case of Rollup and webpack &ndash; only with a standard set of plugins.
+
+## Bundling CKEditor&nbsp;5 with Vite
+
+There is no need for any special configuration to bundle CKEditor&nbsp;5 with recent versions of Vite. Follow the {@link getting-started/quick-start#installing-ckeditor-5-using-npm npm installation guide} to learn more.
+
+## Bundling CKEditor&nbsp;5 with esbuild
+
+Similarly, you need no special configuration to bundle CKEditor&nbsp;5 with recent versions of esbuild. Follow the {@link getting-started/quick-start#installing-ckeditor-5-using-npm npm installation guide} to learn more.
+
+## Bundling CKEditor&nbsp;5 with Rollup
+
+To bundle CKEditor&nbsp;5 with Rollup, you need to install the following plugins:
+
+1. `@rollup/plugin-node-resolve` &ndash; to allow Rollup to resolve dependencies.
+2. `@rollup/plugin-commonjs` &ndash; to convert CommonJS modules to ES6.
+3. `rollup-plugin-import-css` or `rollup-plugin-styles` &ndash; to allow Rollup to handle CSS files.
+4. `@rollup/plugin-json` &ndash; to allow Rollup to handle JSON files.
+5. `@rollup/plugin-terser` &ndash; to minify the output.
```

Comment:
It's a nitpick, but maybe we could order those the same way they are ordered in the snippet below.

Reply:
Done :heavy_check_mark: